examples/strands/strands.bal

```diff
@@ -23,5 +26,7 @@ public function main() {
         io:println("Worker B end");
     }
 
-    io:println("In function worker");
+    // Waiting on named workers until they complete.
```

Comment:
``` suggestion
    // Explicitly waiting on named workers until they complete. 
    // This guarantees that the Ballerina program will not terminate till the workers have completed their execution.
```
Shall we elaborate it more to give the user more understanding of what happens if don't add wait in this case?